shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -378,12 +545,21 @@ - (NSRange)clampSelection:(NSRange)range forText:(NSString*)text {
   return NSMakeRange(start, length);
 }
 
+- (BOOL)isVisibleToAutofill {
+  return self.frame.size.width > 0 && self.frame.size.height > 0;
+}
+
+- (void)setIsVisibleToAutofill:(BOOL)isVisibleToAutofill {
+  self.frame = isVisibleToAutofill ? CGRectMake(0, 0, 1, 1) : CGRectZero;
```

Comment:
This might merit a comment since it seems a bit hacky to change the frame based on something like if it is visible to autofill.